jvb/src/main/java/org/jitsi/videobridge/EndpointMessageTransport.java

```diff
@@ -573,15 +581,16 @@ public BridgeChannelMessage endpointMessage(EndpointMessage message)
             String to = message.getTo();
 
             AbstractEndpoint targetEndpoint = conference.getEndpoint(to);
-            if (targetEndpoint instanceof OctoEndpoint)
+            if (targetEndpoint instanceof Endpoint)
             {
-                targets = Collections.emptyList();
-                sendToOcto = true;
+                targets = Collections.singletonList((Endpoint)targetEndpoint);
+                sendToOcto = false;
             }
             else if (targetEndpoint != null)
             {
-                targets = Collections.singletonList(targetEndpoint);
-                sendToOcto = false;
+                /* TODO: for relayed endpoints, send only to the relevant relay. */
```

Comment:
I think, if we don't insist of going through `Conference`, we can do this with just `((RelayedEndpoint) targetEndpoint).getRelay().sendMessage(message);`